index.js

```diff
@@ -208,14 +211,17 @@ program
 
 program
   .command('build [root] [output]')
+  .option('--baseUrl [baseUrl]', 'overrides baseUrl in site.json, leave argument empty for empty baseUrl')
```

Comment:
`leave argument empty for empty baseUrl`

Maybe just say default value is empty base url